pandas/core/frame.py

```diff
@@ -2215,8 +2215,8 @@ def to_feather(self, path, **kwargs) -> None:
 
         Parameters
         ----------
-        path : str
-            String file path.
+        path : str or file-like object
```

Comment:
can you also add the type to path: FilePathOrBuffer[AnyStr],

Reply:
can you also update in pandas/io/feather_format.py (to_feather)

Reply:
Done